cmd/snap/cmd_list.go

```diff
@@ -39,6 +39,8 @@ type cmdList struct {
 	Positional struct {
 		Snaps []string `positional-arg-name:"<snap>"`
 	} `positional-args:"yes"`
+
+	All bool `short:"a" long:"all"`
```

Comment:
Can we please use either "-a" or "--all", but not both?  I'm personally fine with either. Having both generally leaves me that feeling that we didn't stop to think which form was appropriate, and then delegate to every user the need to understand that they both mean the same.